ipaserver/install/server/__init__.py

```diff
@@ -159,7 +159,6 @@ def domain_level(self, value):
         None,
         description="configure a dogtag KRA",
     )
-    setup_kra = enroll_only(setup_kra)
```

Comment:
Why did you remove this line? It must be here, otherwise `--setup-kra` will appear in `ipa-replica-prepare` once it's ported to the install framework.